apps/core-app/src/utils/constants.ts

```diff
@@ -14,6 +14,55 @@ export const PROPOSAL_FILTERS: { [key: string]: string } = {
 export const DEFAULT_PROPOSAL_PAGE_SIZE = 10;
 export const DEFAULT_MEMBERS_PAGE_SIZE = 25;
 export const DEFAULT_PROPOSAL_SORT: Ordering<Proposal_OrderBy> = {
-  orderBy: 'proposalId',
+  orderBy: 'createdAt',
   orderDirection: 'desc',
 };
+
+export enum ProposalTypeIds {
+  Signal = 'SIGNAL',
+  IssueSharesLoot = 'ISSUE',
+  AddShaman = 'ADD_SHAMAN',
+  TransferErc20 = 'TRANSFER_ERC20',
+  TransferNetworkToken = 'TRANSFER_NETWORK_TOKEN',
+  UpdateGovSettings = 'UPDATE_GOV_SETTINGS',
+  UpdateTokenSettings = 'TOKEN_SETTINGS',
+  TokensForShares = 'TOKENS_FOR_SHARES',
+  GuildKick = 'GUILDKICK',
+}
+
+export const PROPOSAL_TYPE_LABELS: { [key: string]: string } = {
```

Comment:
Record<keyof type ProposalTypeIds, string>

Reply:
having trouble with this.